src/components/tx/DecodedTx/index.tsx

```diff
@@ -119,63 +75,34 @@ const DecodedTx = ({
             expandIcon={<ExpandMoreIcon />}
             className={accordionCss.accordion}
           >
-            <span style={{ flex: 1 }}>Transaction details</span>
-
-            {decodedData
-              ? decodedData.method
-              : tx?.data.operation === OperationType.DelegateCall
-              ? 'Delegate call'
-              : ''}
+            <Box display="flex" alignItems="center">
```

Comment:
This does not look aligned:
![Screenshot 2024-07-25 at 11 35 48](https://github.com/user-attachments/assets/a0c6d931-68aa-4727-b0f3-d76fd3ab55ab)